### DISCUSSION TITLE
Consider testing RTL when there's a UI change

### DISCUSSION THREAD

**Discussion by hishamco** [OrchardCMS/OrchardCore#15535] [General]
I notice that some changes frequently happen on the OC admin site without testing the UI changes in RTL languages, leading to inconsistent UI issues. Let us have a strategy to be applied whenever do a changes in the UI

**Comment by Piedone** [comment]
Perhaps creating some docs on how one can test RTL would be a good start?

**Reply by hishamco** [reply]
Maybe

**Reply by Piedone** [reply]
See: https://github.com/OrchardCMS/OrchardCore/discussions/15800
